src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,157 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Ui handles tasks related to the user interface of Duke. It deals with user interaction, including getting
+ * user input and generating replies from Duke.
+ */
+public class Ui {
+    private Scanner input;
+
+    /**
+     * Constructs a Ui object. A Scanner reading in from System.in will be instantiated for user input.
+     */
+    public Ui() {
+        this.input = new Scanner(System.in);
+    }
+
+    /**
+     * Returns a loading error due to Duke being unable to load in saved tasks from the file specified. Informs
+     * the user that a new file will be created.
+     * 
+     * @return Loading error string.
+     */
+    public String showLoadingError() {
+        return "Error: Unable to load saved tasks from file.\n"
+                + "New data file will be created.";
+    }
+
+    /**
+     * Returns the Duke logo and greeting.
+     * 
+     * @return Greeting string.
+     */
+    public String showGreeting() {
+        String logo = " ____        _        \n"
```

Comment:
Strings like this one can be stored together in a `Message` file as `public final static String`.

Reply:
Another possible example is the dividing line "----------------------";